examples/azure-rules.yaml

```diff
@@ -656,4 +656,18 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check OS and package versions are supported and up-to-date"
+    checks:
+      - id: 1.6.1
+        description: "Check the base product"
+        audit: 'ls -la /etc/products.d/baseproduct | sed "s/^.*-> //"'
```

Comment:
this is scary code, better do something like 

`readlink /etc/products.d/baseproduct`